cognite/neat/_rules/_constants.py

```diff
@@ -106,13 +106,9 @@ def get_reserved_words(key: Literal["class", "view", "property", "space"]) -> li
 DMS_PROPERTY_ID_COMPLIANCE_REGEX = (
     rf"(?!^({'|'.join(get_reserved_words('property'))})$)" r"(^[a-zA-Z][a-zA-Z0-9_]{0,253}[a-zA-Z0-9]?$)"
 )
```

Comment:
reduced restriction on class and property ids for information architect to allow, but then will be caught later on